### ISSUE TITLE
Question. Can Markdown Editor be used on installations where OnlyOffice is deployed

### ISSUE THREAD

**Issue by Daksol** [icewind1991/files_markdown#156]
We have NC18 with OnlyOffice. The Markdown Editor is shown as being installed in the Apps list. However we get a very basic markdown editor for text files - which are saved with the MD extension.

Possible to have Markdown Editor used for text files when OnlyOffice is in use.

Thanks

**Comment by fenvarien** [comment]
We have the same problem, Markdown Editor is somehow blocked by OnlyOffice. Is there any known workaround? (NC20, OnlyOffice 6.1)

**Comment by baslow** [comment]
Yes. I've just installed Nextcloud with OnlyOffice. Although Markdown installs I cannot seem to access it. If I disable Text I don't see any way to edit .md files.

**Comment by Daksol** [comment]
I think there may be an issue wrt how well certain NextCloud Apps related to editing text files "play nicely together"

So that may include:-

* ONLYOFFICE/onlyoffice-nextcloud
* NextCloud/files_markdown
* NextCloud/Text
* NextCloud/files_texteditor

What is best place to raise this sort of issue? 

@icewind1991 can you suggest the appropriate process here.....

Dank u wel

**Comment by baslow** [comment]
I now have Nextcloud 21.0.0 installed. With both Markdown Editor and Plain Text Editor installed and enabled alongside OnlyOffice I  still cannot get  wither Markdown  nor Plain Text to work. The three-dot menu inside the file simply doesn't offer the option.

**Comment by archont00** [comment]
Nextcloud 23:
- "Text" app disabled.
- Plain Text Editor and Mark Down Editor enabled.
- OnlyOffice installed (own installation: community version).

It works as expected - txt and md files are open in Plain Text Editor and Mark Down Editor, respectively. No need to use "..." menu.

AFAIK it worked also with "Text" installed - via "..." menu.

Have you checked (Admin) Settings // ONLYOFFICE and disabled "txt" extension so that it does not open in OO?
